readyapi/build.gradle

```diff
@@ -1,6 +1,6 @@
 dependencies {
     compile project(':soapui')
-    compileOnly 'com.smartbear:ready-api-soapui-pro:2.3.0'
+    compileOnly 'com.smartbear.soapui:soapui:5.4.0'
```

Comment:
Why that changed? What's wrong with ready-api dependency?

Reply:
ready-api dependency version set to 2.6.0